manifests/init.pp

```diff
@@ -54,6 +56,8 @@
   $proxy_username = $katello::params::proxy_username,
   $proxy_password = $katello::params::proxy_password,
   $cdn_ssl_version = $katello::params::cdn_ssl_version,
+
+  $packages = $katello::params::packages
```

Comment:
Why introduce a separate parameter when there's already one in params that you could expose here?